src/resources/Pipelines/MLAssociations/MLAssociations.ts

```diff
@@ -0,0 +1,57 @@
+import API from '../../../APICore';
+import Resource from '../../Resource';
+import Pipelines from '../Pipelines';
+import {
+    AssociatedPipelinesData,
+    AssociationsListModel,
+    CreateAssociation,
+    EditAssociation,
+    ListAssociationsParams,
+    MLAssociationModel,
+} from './MLAssociationsInterfaces';
+
+export default class MLAssociations extends Resource {
+    static getBaseUrl = (pipelineId: string) => `/rest/search/v2/admin/pipelines/${pipelineId}/ml/model/associations`;
+
+    list(pipelineId: string, options?: ListAssociationsParams) {
+        return this.api.get<AssociationsListModel>(this.buildPath(MLAssociations.getBaseUrl(pipelineId), options));
+    }
+
+    associate(pipelineId: string, options: CreateAssociation) {
+        return this.api.post(`${MLAssociations.getBaseUrl(pipelineId)}?organizationId=${API.orgPlaceholder}`, options);

```

Comment:
You can use the `buildPath` method to safely add query string parameters to the url
```suggestion
        return this.api.post(this.buildPath(MLAssociations.getBaseUrl(pipelineId), {organizationId: this.api.organizationId})`, options);
```